README.rst

```diff
@@ -41,9 +41,9 @@ urllib3 is powerful and easy to use::
 Installing
 ----------
 
-urllib3 can be installed with `pip <https://pip.pypa.io>`_::
+urllib3 can be installed with `Pip <https://pip.pypa.io>`_::
```

Comment:
Any reason to change from pip to Pip? I've never seen the second one

Reply:
Ah okay, I'll have to fix this in another place too.